tests/config_class/test_config_load_robusta_ai.py

```
@@ -110,3 +110,22 @@ def test_server_not_loads_robusta_ai_when_model_var_exists_and_false_env_var(
+
+ROBUSTA_HOLMES_ARGS_MODELS = RobustaModelsResponse(
+    models=["Robusta/test", "Robusta/sonnet-1m"],
+    models_holmes_args={"Robusta/sonnet-1m": {"max_context_size": 1000000}},
+    default_model=None,
+)
+
+
+@patch("holmes.core.llm.ROBUSTA_AI", True)
+@patch("holmes.core.llm.fetch_robusta_models", return_value=ROBUSTA_HOLMES_ARGS_MODELS)
+@patch("holmes.config.Config._Config__get_cluster_name", return_value="test")
+def test_robusta_ai_config_get_llm_context_override(mock_cluster, *, monkeypatch):
```

Ruff (0.12.2)

Unused function argument: `mock_cluster`

(ARG001)